requirements-vllm-cuda.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+# Dependencies for installing vLLM on CUDA
+
+# vLLM only supports Linux platform (including WSL)
+vllm>=0.5.0; sys_platform == 'linux' and platform_machine == 'x86_64' and python_version >= '3.9' and python_version <= '3.11'
```

Comment:
You also have to include the dependencies from https://github.com/vllm-project/vllm/blob/main/requirements-cuda.txt (except for Ray). At least `vllm-flash-attn` should be installed to improve performance.

Reply:
Yes thanks, I was just testing out something on the runner first, I'll include these.